query-engine/connector-test-kit-rs/query-engine-tests/tests/new/interactive_tx.rs

```diff
@@ -86,6 +86,7 @@ mod interactive_tx {
 
         let error = res.err().unwrap();
         let known_err = error.as_known().unwrap();
+        println!("KNOWN ERROR {:?}", known_err);
```

Comment:
I've had a couple flaky MySql 5.7 tests fail here. I cannot reproduce locally. I'm adding this here so I can see what the error being thrown back is.